src/java.desktop/share/classes/sun/awt/image/ImagingLib.java

```diff
@@ -90,20 +88,14 @@ public static native int lookupByteRaster(Raster src, Raster dst,
 
     static {
 
-        PrivilegedAction<Boolean> doMlibInitialization =
-            new PrivilegedAction<Boolean>() {
-                public Boolean run() {
-                    try {
-                        System.loadLibrary("mlib_image");
-                    } catch (UnsatisfiedLinkError e) {
-                        return Boolean.FALSE;
-                    }
-                    boolean success = init();
-                    return Boolean.valueOf(success);
-                }
-            };
+        boolean success = false;
+        try {
+            System.loadLibrary("mlib_image");
+            success = init();
+        } catch (UnsatisfiedLinkError e) {
+        }
 
-        useLib = AccessController.doPrivileged(doMlibInitialization);
+        useLib = Boolean.valueOf(success);
```

Comment:
```suggestion
        useLib = success;
```
Unnecessary wrapping into `Boolean`.